### PR TITLE
Add scumm vm logo

### PR DESCRIPTION
**Pull request by ayushbisht2001**
I have added the ScummVm logo in README.md along with a ScummVmlogo.png file. I am not a hundred percent sure but this might not show up on other sites (like pypi). Because I had also updated the README.md  file of Pygame as they were also facing the same issues . For that you have to merged this PR , becoz I have added ScummVmLogo.png file. Then , using that image I can easily add the external link of that image in README.md. Hope you like my work.

### CODE REVIEW

**Comment by ayushbisht2001** [comment]
@Mataniko , Please suggest me , if I am doing anything wrong .

**Comment by sev-** [comment]
Please follow our commit guidelines.

**Comment by ayushbisht2001** [comment]
@sev-  sorry for that , now I have deleted one file.

**Comment by bluegr** [comment]
Wait, now I'm confused. You got 2 commits which add a logo to the README file, and then you deleted the README file? Why?

**Comment by ayushbisht2001** [comment]
@bluegr , As per guidelines commits shouldn't contain multiple unrelated changes. That's why I feel to make separate PR for that. Sorry for the Inconveniency , If I am doing wrong then please suggest me how to do it.

**Comment by criezy** [comment]
Deleting the README.md file is definitely wrong. Please drop that commit.

I think the comment from sev- is about the formatting of the commit message that is wrong.
The first line should start with a subsystem name, in this case `README`.
You can see details here: https://wiki.scummvm.org/index.php?title=Commit_Guidelines
And you can look at past commit messages for this file for examples: https://github.com/scummvm/scummvm/commits/master/README.md

**Comment by Mataniko** [comment]
Please also see the feedback from your previously closed PR: https://github.com/scummvm/scummvm/pull/2733 and avoid opening duplicate PRs and instead fix the existing one.

File: README.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
-# [ScummVM README](https://www.scummvm.org/) · [![Build Status](https://travis-ci.org/scummvm/scummvm.svg?branch=master)](https://travis-ci.org/scummvm/scummvm) ![CI](https://github.com/scummvm/scummvm/workflows/CI/badge.svg) [![Translation status](https://translations.scummvm.org/widgets/scummvm/-/scummvm/svg-badge.svg)](https://translations.scummvm.org/engage/scummvm/?utm_source=widget) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](CONTRIBUTING.md#pull-requests) [![Codacy Badge](https://app.codacy.com/project/badge/Grade/e06e5b18f8464fef859b5a7f78d10357)](https://www.codacy.com/gh/scummvm/scummvm/dashboard?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=scummvm/scummvm&amp;utm_campaign=Badge_Grade)
+
+![ScummVm](ScummVmlogo.png)
```
Comment (Mataniko) [inline README.md:2]: do not include a logo file in the repo, see notes in the old PR.

**Review by Mataniko — CHANGES_REQUESTED** [review]
Please see https://wiki.scummvm.org/index.php?title=Commit_Guidelines

**Comment by sev-** [comment]
Anyway, this PR is pointless. Closing.

**Pull request closed without merging**